tests/unit/src/test/scala/tests/decorations/SyntheticDecorationsLspSuite.scala

```diff
@@ -696,4 +696,64 @@ class SyntheticDecorationsLspSuite extends BaseLspSuite("implicits") {
       )
     } yield ()
   }
+
+  test("issue-3938") {
```

Comment:
Suggestion: maybe paste link to issue next to test and give it human-readable name like `for-yield`?